shopfloor_mobile_packing/static/src/js/i18n/en.json

```
@@ -0,0 +1,6 @@
+{
+  "cluster_picking.pack_picking_put_in_pack.title": "Packing: Enter the number of boxes used.",
+  "cluster_picking.pack_picking_put_in_pack.scan_placeholder": "Number of boxes used",
```

As now we choose a package type, this needs to be adapted.
```suggestion
  "cluster_picking.pack_picking_put_in_pack.title": "Packing",
  "cluster_picking.pack_picking_put_in_pack.scan_placeholder": "Package type",
```